typescript/audio-summarizer/README.md

```diff
@@ -0,0 +1,188 @@
+# Emojifier 🤖🎨 - A simple audio summarizer with observability on Literal AI
+
+In this cookbook, we will start with a simple audio summarizer using the Web Speech API and Next.js. We will then add observability to the application using Literal AI.
+
+The base application converts audio to text and summarize the text as a sequence of emojis. It consists of :
+
+- A web page with a simple UI to record audio, using the [AudioRecorder](https://www.npmjs.com/package/react-audio-voice-recorder) React component.
+- A Next.js API route to convert the audio to text using OpenAI's [Whisper API](https://openai.com/index/whisper/).
+- A Next.js API route to summarize the text using OpenAI's [GPT-4](https://openai.com/index/gpt-4/).
+
+You can test the base application by running the development server. The only pre-requisite is to have Node.js installed on your machine.
+
+```bash
+cd without-literal
+npm install
+npm run dev
+```
+
+## Prerequisites
+
+Before we get started, you will need to create a free account on [Literal AI](https://cloud.getliteral.ai/). You will also need to create a new project and generate an API key from your project's "Settings > General" section.
+
+For the OpenAI calls to work, you will also need to generate an OpenAI API key from the [OpenAI API](https://platform.openai.com/api-keys) platform.
+
+You can now copy the provided `.env.example` file to a new `.env` file and fill in the required environment variables.
+
+## Adding Literal AI to the application
+
+We will start by adding the Literal AI SDK to the application. You can install the SDK using npm:
+
+```bash
+npm install @literalai/client
+```
+
+## Decoupling the prompt from our code
+
+Having prompts in the code can get unwieldy with time, those big templated strings get hard to read and maintain (although YMMV on this). For this project I have opted to manage them entirely on Literal AI, which allows me to iterate on the prompt's messages and settings without having to redeploy my application.
+
+I have created a new prompt on Literal AI with the following steps :
+
+- Navigate to my project on Literal AI
+- Click on the "Prompts" tab
+- Click on the "New Prompt" button
+- Click on the "+ Message" button in the "Template" section
+- Copy my prompt from the code to the editor
+- Adjust models, temperatures and other settings as needed
+- Save the prompt with the name "Emojifier Prompt" (be sure to copy this exact name as it will be used to retrieve the prompt through the API)
+
+Now I will edit the `src/app/api/emojify/route.ts` file and add the following :
```

Comment:
Instead of saying the user to edit the file, we can instead showcase him a ready to use codebase.

Instead of:
```
Now I will edit the `src/app/api/emojify/route.ts` file and add the following :
```

Having:
```
You can see the code that retrieves the prompt from Literal AI in `src/app/api/emojify/route.ts`
```

Reply:
hmmm Ok, I dind't undertand at first that you are guiding the user on how you modified the app codebase to add Literal AI observability features. If so, please discard by previous comment.

Reply:
Great feedback, thanks ! I've clarified by changing the wordings to "You can create a new [prompt]" & "Now you can edit the ..." it is indeed much clearer as to the intention of the tutorial.